src/http.rs

```diff
@@ -0,0 +1,22 @@
+use curl;
+use oauth2::*;
+use app::App;
+
+pub fn github(app: &App, url: &str, auth: &Token)
```

Comment:
The response-code handling isn't uniform, though. In particular when querying team membership, 404 is how Github reports "false". Do you want this to take a closure for (maybe) processing the error code?